### ISSUE TITLE
/proc/self/mountinfo is not the same under LX than under a real linux kernel

### ISSUE THREAD

**Issue by sjorge** [joyent/illumos-joyent#106]
CentOS 7 on HW:

```
cat /proc/self/mountinfo 
17 59 0:16 / /sys rw,nosuid,nodev,noexec,relatime shared:6 - sysfs sysfs rw
18 59 0:3 / /proc rw,nosuid,nodev,noexec,relatime shared:5 - proc proc rw
19 59 0:5 / /dev rw,nosuid shared:2 - devtmpfs devtmpfs rw,size=12251544k,nr_inodes=3062886,mode=755
20 17 0:15 / /sys/kernel/security rw,nosuid,nodev,noexec,relatime shared:7 - securityfs securityfs rw
21 19 0:17 / /dev/shm rw,nosuid,nodev shared:3 - tmpfs tmpfs rw
22 19 0:11 / /dev/pts rw,nosuid,noexec,relatime shared:4 - devpts devpts rw,gid=5,mode=620,ptmxmode=000
23 59 0:18 / /run rw,nosuid,nodev shared:21 - tmpfs tmpfs rw,mode=755
24 17 0:19 / /sys/fs/cgroup ro,nosuid,nodev,noexec shared:8 - tmpfs tmpfs ro,mode=755
25 24 0:20 / /sys/fs/cgroup/systemd rw,nosuid,nodev,noexec,relatime shared:9 - cgroup cgroup rw,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd
26 17 0:21 / /sys/fs/pstore rw,nosuid,nodev,noexec,relatime shared:19 - pstore pstore rw
27 24 0:22 / /sys/fs/cgroup/hugetlb rw,nosuid,nodev,noexec,relatime shared:10 - cgroup cgroup rw,hugetlb
28 24 0:23 / /sys/fs/cgroup/perf_event rw,nosuid,nodev,noexec,relatime shared:11 - cgroup cgroup rw,perf_event
29 24 0:24 / /sys/fs/cgroup/freezer rw,nosuid,nodev,noexec,relatime shared:12 - cgroup cgroup rw,freezer
30 24 0:25 / /sys/fs/cgroup/devices rw,nosuid,nodev,noexec,relatime shared:13 - cgroup cgroup rw,devices
31 24 0:26 / /sys/fs/cgroup/memory rw,nosuid,nodev,noexec,relatime shared:14 - cgroup cgroup rw,memory
32 24 0:27 / /sys/fs/cgroup/cpu,cpuacct rw,nosuid,nodev,noexec,relatime shared:15 - cgroup cgroup rw,cpuacct,cpu
33 24 0:28 / /sys/fs/cgroup/cpuset rw,nosuid,nodev,noexec,relatime shared:16 - cgroup cgroup rw,cpuset
34 24 0:29 / /sys/fs/cgroup/blkio rw,nosuid,nodev,noexec,relatime shared:17 - cgroup cgroup rw,blkio
35 24 0:30 / /sys/fs/cgroup/net_cls rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,net_cls
56 17 0:31 / /sys/kernel/config rw,relatime shared:20 - configfs configfs rw
59 1 253:1 / / rw,relatime shared:1 - xfs /dev/mapper/vg_facl1-lv_root rw,attr2,inode64,noquota
15 18 0:14 / /proc/sys/fs/binfmt_misc rw,relatime shared:22 - autofs systemd-1 rw,fd=30,pgrp=1,timeout=300,minproto=5,maxproto=5,direct
36 19 0:33 / /dev/hugepages rw,relatime shared:23 - hugetlbfs hugetlbfs rw
38 17 0:7 / /sys/kernel/debug rw,relatime shared:24 - debugfs debugfs rw
37 19 0:13 / /dev/mqueue rw,relatime shared:25 - mqueue mqueue rw
39 15 0:34 / /proc/sys/fs/binfmt_misc rw,relatime shared:26 - binfmt_misc binfmt_misc rw
41 18 0:35 / /proc/fs/nfsd rw,relatime shared:27 - nfsd nfsd rw
74 59 8:1 / /boot rw,relatime shared:28 - ext3 /dev/sda1 rw,data=ordered
73 59 253:2 / /var rw,relatime shared:29 - xfs /dev/mapper/vg_facl1-lv_var rw,attr2,inode64,noquota
77 73 0:36 / /var/lib/nfs/rpc_pipefs rw,relatime shared:30 - rpc_pipefs sunrpc rw
79 73 0:37 / /var/lib/ctdb rw,relatime shared:31 - tmpfs tmpfs rw,size=2097152k,mode=700
81 59 0:39 / /shared/aclfiler1 rw,noatime shared:32 - gpfs /dev/aclfiler1 rw
85 23 0:41 / /run/user/506 rw,nosuid,nodev,relatime shared:34 - tmpfs tmpfs rw,size=2452464k,mode=700,uid=506,gid=1000
83 23 0:40 / /run/user/0 rw,nosuid,nodev,relatime shared:33 - tmpfs tmpfs rw,size=2452464k,mode=700
```

LX

```
15 0 203:1 / / rw - zfs /dev/zfsds0 rw
16 15 90:65540 / /data rw - lofs /data rw
17 15 90:65560 / /mm/anime ro - lofs /mm/anime ro
18 15 90:65561 / /mm/movies ro - lofs /mm/movies ro
19 15 90:65562 / /mm/music ro - lofs /mm/music ro
20 15 90:65563 / /mm/series/archive ro - lofs /mm/series/archive ro
21 15 90:65574 / /mm/series/live ro - lofs /mm/series/live ro
31 15 571:0 / /dev rw - devtmpfs devtmpfs rw
38 15 572:0 / /proc rw - proc proc rw
40 15 573:0 / /sys rw - sysfs sysfs rw
41 15 559:10 / /dev/shm rw - tmpfs tmpfs rw
42 15 559:11 / /run rw - tmpfs tmpfs rw
43 15 559:22 / /data/plex/Logs rw - tmpfs tmpfs rw
44 15 0:1 / /native/usr ro - zfs /native/usr ro
```

All the mount options after ro or rw are missing

**Comment by sjorge** [comment]
@jjelinek didn’t this get fixed recently?

**Comment by jjelinek** [comment]
There was a recent fix here for OS-6475, commit 0d5719a2. I'm not certain if that fixes this or not.

**Comment by sjorge** [comment]
The error I was hitten in salt due to this now looks to be fixed.
